tutor/src/models/annotations/ux.js

```diff
@@ -0,0 +1,13 @@
+import {
+  computed, observable, action,
+} from 'mobx';
+
+export default class AnnotatorUX {
+
+  @observable isSummaryVisible = false;
+
+  @action.bound toggleSummary() {
+    console.log("TOGGLE SUM")
```

Comment:
Do you want this left in?

Reply:
Nope, that was a debug thing that got left in 😦   I removed in #1978